src/tests/taskmanager_test.py

```diff
@@ -0,0 +1,84 @@
+"""
+Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import time
+import sys
+import os
+import unittest
```

Comment:
I noticed you don't use unittest anywhere. Does this code still run if you remove this import?

Reply:
TestTaskManager inherits from unittest.TestCase

Reply:
Sounds good.